src/bin.ts

```diff
@@ -64,6 +65,9 @@ function isTrue (value: string) {
 
 const cwd = argv.cwd ? resolve(argv.cwd) : process.cwd()
 const emitter: Emitter = new EventEmitter()
+if (argv.loglevel) {
```

Comment:
Can you move this `if` statement to it's own section - either above or below this block instead of crammed in the middle? 😄